### ISSUE TITLE
Error: The 'cache' field on 'RequestInitializerDict' is not implemented.

### ISSUE THREAD

**Issue by 0xthierry** [AssemblyAI/assemblyai-node-sdk#57]
When using the following method

```ts
const transcript = await client.transcripts.transcribe(data, options);
```

In my Cloudflare Worker I have the following error:

```
TryResult {
  data: null,
  error: Error: The 'cache' field on 'RequestInitializerDict' is not implemented.
      at checkURL (file:///Users/0xthierry/Workspace/Company/project/.wrangler/tmp/bundle-ML7h0E/checked-fetch.js:9:9)
      at Object.apply (file:///Users/0xthierry/Workspace/Company/project/.wrangler/tmp/bundle-ML7h0E/checked-fetch.js:27:3)
      at FileService.<anonymous> (file:///Users/0xthierry/Workspace/Company/project/node_modules/.pnpm/assemblyai@4.4.6/node_modules/assemblyai/dist/index.mjs:117:36)
      at Generator.next (<anonymous>)
      at null.<anonymous> (file:///Users/0xthierry/Workspace/Company/project/node_modules/.pnpm/assemblyai@4.4.6/node_modules/assemblyai/dist/index.mjs:38:71)
      at new Promise (<anonymous>)
      at __awaiter (file:///Users/0xthierry/Workspace/Company/project/node_modules/.pnpm/assemblyai@4.4.6/node_modules/assemblyai/dist/index.mjs:34:12)
      at FileService.fetch (file:///Users/0xthierry/Workspace/Company/project/node_modules/.pnpm/assemblyai@4.4.6/node_modules/assemblyai/dist/index.mjs:100:16)
      at FileService.<anonymous> (file:///Users/0xthierry/Workspace/Company/project/node_modules/.pnpm/assemblyai@4.4.6/node_modules/assemblyai/dist/index.mjs:139:41)
      at Generator.next (<anonymous>),
  [Symbol(Symbol.iterator)]: [GeneratorFunction (anonymous)]
}
```

It's an API running on Cloudflare Workers using Hono as an API router.

package.json

```json
"assemblyai": "^4.4.6",
```

**Comment by 0xthierry** [comment]
Probably something that changed in this release

https://github.com/AssemblyAI/assemblyai-node-sdk/releases/tag/v4.4.2

**Comment by 0xthierry** [comment]
The version v4.4.1 is working

**Comment by Swimburger** [comment]
This was a patch we made to disable caching using an extra option.
It looks like Cloudflare Workers doesn't like that we specify this option.

For now, I'd stick to the version that works. We'll figure out a way to not specify this option specifically for CF Workers.

**Comment by Swimburger** [comment]
I released `4.4.7-beta.0` which adds a separate build for Cloudflare Workers (`workerd` export).
This separate build removes the cache option from fetch requests, only for Cloudflare Workers.
We'll add this cache option back when Cloudflare Workers finally fixes this bug.

For this to land in the next minor version (without beta), the change needs to go through internal review.
So keep an eye out for more releases.

**Comment by Swimburger** [comment]
The fix has been released with 4.4.7 just now. Let us know if you have any other issues.

**Comment by 0xthierry** [comment]
Thank you @Swimburger
